app/models/service_ansible_playbook.rb

```diff
@@ -63,15 +63,32 @@ def on_error(action)
 
   def manageiq_extra_vars(action)
     {
-      'api_url'   => MiqRegion.my_region.remote_ws_url,
-      'api_token' => Api::UserTokenService.new.generate_token(evm_owner.userid, 'api'),
-      'service'   => href_slug,
-      'user'      => evm_owner.href_slug,
-      'group'     => miq_group.href_slug,
-      'action'    => action
+      'api_url'     => api_url,
+      'api_token'   => api_token,
+      'service'     => href_slug,
+      'user'        => evm_owner.href_slug,
+      'group'       => miq_group.href_slug,
+      'action'      => action,
+      'X_MIQ_Group' => evm_owner.current_group.description
     }.merge(request_options_extra_vars)
   end
 
+  def manageiq_connection_env
```

Comment:
Is there a way for us to share this method with [automate](https://github.com/ManageIQ/manageiq-automation_engine/blob/b2ba90e6cbcdd033fbf3bcb0940e8c2ea958434e/lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_playbook_method.rb#L112-L129)